icaruscode/Decode/DaqDecoderICARUSTPCwROI_module.cc

```diff
@@ -548,16 +546,21 @@ void DaqDecoderICARUSTPCwROI::processSingleFragment(size_t
             continue;
         }
 
-        const icarusDB::ChannelPlanePairVec& channelPlanePairVec = fChannelMap->getChannelPlanePair(boardIDVec[board]);
-
         uint32_t boardSlot = physCrateFragment.DataTileHeader(board)->StatusReg_SlotID();
 
+        const icarusDB::ChannelPlanePairVec& channelPlanePairVec = fChannelMap->getChannelPlanePair(boardIDVec[boardSlot]);
+
         std::stringstream outputString;
         outputString << "********************************************************************************\n"
-                     << "FragmentID: " << std::hex << fragmentID << ", Crate: " << crateName << std::dec << ", boardID: " << boardSlot << "/" << nBoardsPerFragment << ", size " << channelPlanePairVec.size() << "/" << nChannelsPerBoard;
+                     << "FragmentID: " << std::hex << fragmentID << std::dec << ", Crate: " << crateName << ", boardID: " << boardSlot << "/" << nBoardsPerFragment << ", size " << channelPlanePairVec.size() << "/" << nChannelsPerBoard;
 
```

Comment:
Good point, future upgrade. Generally, I am not sure I like how the message service formats unless I make a big string... I can get more condense output with std::cout. But I suppose that reflects my lack of knowledge on tools that may be available in the message service.

Reply:
I think that under the hood MessageLogger objects like `mf::LogDebug` pass everything to a `std::ostream`. I don't remember having had any problem with any I/O manipulator, standard or otherwise.

Reply:
I think the problem is that each call to an mf stream results in a new line that begins with a label...

Reply:
Sorry, I was looking at something else, I realize the above suggestion is correct.